app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SessionDetailActivity.java

```diff
@@ -13,10 +14,14 @@
 public class SessionDetailActivity extends BaseActivity {
 
     private static final String EXTRA_SESSION_ID = "session_id";
+    private static final String EXTRA_PARENT_NAME = "parent_name";
+
+    private String parentName;
 
     public static Intent createIntent(@NonNull Context context, int sessionId) {
         Intent intent = new Intent(context, SessionDetailActivity.class);
         intent.putExtra(EXTRA_SESSION_ID, sessionId);
+        intent.putExtra(EXTRA_PARENT_NAME, context.getClass().getCanonicalName());
```

Comment:
Sorry to be late 🙇 
I guess boolean value is better to check 🤔 
I'm not sure but if there is better way, maybe someone will improve.
Can you try to fix as you want?